CMakeLists.txt

```diff
@@ -70,18 +70,27 @@ else()
         GIT_REPOSITORY https://github.com/apple/swift-foundation-icu.git
         GIT_TAG 0.0.9)
 endif()
+FetchContent_MakeAvailable(SwiftFoundationICU)
```

Comment:
`FetchContent_MakeAvailable` should really be invoked once at the end as per the documentation.